cmd/sti/main.go

```diff
@@ -126,7 +130,7 @@ func newCmdCreate() *cobra.Command {
 				cmd.Help()
 				os.Exit(1)
 			}
-			b := sti.NewCreate(args[0], args[1])
+			b := create.NewCreate(args[0], args[1])
```

Comment:
I agree with create.New() but I don't like strategies.New() as we are not just returning a strategy but we are making decisions about what strategy is going to be returned. The object returned is a Builder, so GetBuilder kinda makes sense to me.